src/car/dto/update-car.dto.ts

```diff
@@ -0,0 +1,51 @@
+import { CajaType } from '@prisma/client';
+import {
+  IsBoolean,
+  IsEnum,
+  IsInt,
+  IsOptional,
+  IsPositive,
+  IsString,
+  Max,
+  Min,
+} from 'class-validator';
+
+export class UpdateCarDto {
```

Comment:
If the only difference between CreateCarDto and UpdateCarDto is the use of `@IsNotEmpty`/`@IsOptional`, i think that you can use a mapped-type instead https://docs.nestjs.com/openapi/mapped-types#partial
```suggestion
export class UpdateCarDto extends PartialType(CreateCarDto)  {}
```

Reply:
I didn't know about that feature, very practical.
I tested it locally and all the validations work as well.